shared/src/commonMain/kotlin/org/hyperskill/app/problems_limit/presentation/ProblemsLimitFeature.kt

```diff
@@ -4,7 +4,7 @@ import org.hyperskill.app.subscriptions.domain.model.Subscription
 import kotlin.time.Duration
 
 object ProblemsLimitFeature {
-    internal sealed interface State {
```

Comment:
Why `internal` modifier removed? We have public `ViewState`

Reply:
Sorry, forgot to rollback this change.
I was a local experiment with substate.